docs/development/scripts.rst

```diff
@@ -5,40 +5,22 @@ Writing Command-Line Scripts
 Command-line scripts in Astropy should follow a consistent scheme to promote
 readability and compatibility.
 
-The actual script should be in the ``/scripts`` directory of the Astropy
-source distribution, and should do nothing aside from importing a ``main``
-function from astropy and execute it.  This was partly necessary because the
-"2to3" utility that converted python 2.x code to 3.x does not convert scripts.
-These scripts should be executable, include ``#!/usr/bin/env python`` at the
-top, and should *not* end in ``.py``.
-
-The ``main`` functions these scripts call should accept an optional single
-argument that holds the ``sys.argv`` list, except for the script name
-(e.g., ``argv[1:]``). This function can live in its own module, or be part of a
-larger module that implements a class or function for astropy library use. The
-``main`` function should do very little actual work - it should only parse the
-arguments and pass those arguments on to some library function so that the
-library function can be used programmatically when needed.
-Command-line options can be parsed however desired, but the :mod:`argparse`
-module is recommended when possible, due to its simpler and more flexible
-interface relative to the older :mod:`optparse`. :mod:`argparse` is only
-available in python >=2.7 and >=3.2, however, so it should be imported as
-``from astropy.util.compat import argparse`` .
-
+Setuptools' `"entry points"`_ are used to automatically generate wrappers with
+the correct extension. The scripts can live in their own module, or be part of
+a larger module that implements a class or function for astropy library use.
+They should have a ``main`` function to parse the arguments and pass those
+arguments on to some library function so that the library function can be used
```

Comment:
I thought it was helpful previously to mention that the argument to ``main`` should be a single argument that will contain ``sys.argv[1:]`` - can you add this back? Also, it would be good to mention that the entry point will be added to ``setup.py`` (I know you show it below in the example, but maybe you can already mention it here and refer to the example).